src/Solucion_JavierRamiro/src/programacion/empresaprogramacion/Gerente.java

```diff
@@ -0,0 +1,23 @@
+package programacion.empresaprogramacion;
+
+public class Gerente extends Empleado {
+  private String departamento;
+
+  public Gerente(String nombre, double sueldo, String departamento2) {
+      super(nombre, sueldo);
+  }
+
+  public String getDepartamento() {
+      return departamento;
+  }
+
+  @Override
+  public TipoEmpleado getTipo() {
+      return TipoEmpleado.GERENTE;
+  }
+
+  @Override
+  public String toString() {
+      return "Gerente [nombre=" + getNombre() + ", sueldo=" + getSueldo() + ", departamento=" + departamento + "]";
```

Comment:
Dejar los nombres de los atributos no aporta información. La cadena que devuelve `toString()`, cuando más breve y concisa, mejor: 

```suggestion
      return "Gerente:" + getNombre() + ", " + getSueldo() + "€/año, " + departamento;
```